data/reference/syntax.adoc

```diff
@@ -0,0 +1,267 @@
+= AsciiDoc Syntax Quick Reference - an overview of the AsciiDoc syntax.
+
+== Paragraphs
+
+A normal paragraph.
+
+  An indented literal
+  paragraph.
+
+== Formatted Text
+
+*bold text*.
+_emphasized text_.
+`monospaced text`.
+
+=== Unconstrained
+
+In short, unconstrained means anywhere, including within a word:
+
+bold ** text with spaces **.
+emphasized le__tt__ers.
+m``o``n``o``spaced letters.
+
+== Document Links
+
+// define link target
+[[id]]
+// link to target id
+<<id,caption>>
+// link to an external file
+link:file#id[caption]
+
+=== URLs
+
+Use normal URL and email address syntax or:
+
+mailto:hello@domain.org[Say hello!]
+link:https://domain.org/page[Go to page]
+link:../path/to/file.txt[Read file]
+link:\\path\to\file.pdf[Download file.pdf]
+
+== Lists
+
+=== Unordered
+
+* level 1
+** level 2
+*** level 3
+**** level 4
+***** level 5
+* level 1
+
+- Edgar Allen Poe
+- Sheri S. Tepper
+- Bill Bryson
+
+=== Ordered
+
+. Step 1
+. Step 2
+.. Step 2a
+.. Step 2b
+. Step 3
+
+=== Checklist
+
+* [*] checked
+* [x] also checked
+* [ ] not checked
+*     normal list item
+
+=== Description
+
+first term:: definition of first term
+second term:: definition of second term
+
+== Document Structure
+
+=== Document Header
+
+= My Document's Title
+Doc Writer <doc.writer@domain.org>
+v1.0, 2019-01-01
+
+My document provides...
+
+=== Section titles
+
+= Document Title (Level 0)
+== Level 1 Section Title
+=== Level 2 Section Title
+==== Level 3 Section Title
+===== Level 4 Section Title
+====== Level 5 Section Title
+== Another Level 1 Section Title
+
+=== Include files
+
+include::basics.adoc[]
+include::https://domain.org/installation.adoc[]
+
+== Delimited Blocks
+
+////
+Comment
+////
+
+[NOTE]
+// one of: NOTE, TIP, WARNING, IMPORTANT, CAUTION
+====
+admonition
+====
+
+[example]
+====
+example
+====
+
+```language
+fenced
+```
+
+[listing]
+----
+listing
+----
+
+[literal]
+....
+literal
+....
+
+--
+open
+--
+
+[pass]
+++++
+passthrough
+++++
+
+[quote, attribution, citation title and information]
+____
+quote
+____
+
+[sidebar]
+****
+sidebar
+****
+
+[source, language, options]
+----
+source
+----
+
+[stem]
+++++
+stem
+++++
+
+|===
```

Comment:
Leave the table example for it's own section.

Reply:
Ok